cmd/issuer-rest/startcmd/start.go

```diff
@@ -470,6 +470,12 @@ func startIssuer(parameters *issuerParameters) error { //nolint:funlen
 	router.PathPrefix("/flightbooking").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		http.ServeFile(w, r, "static/flightbooking.html")
 	})
+	router.PathPrefix("/applygreencard").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
probably we can rename this to standard name ie, `PRC` instead of `GreeenCard`.

Reply:
okay will do